tests/js/specs/row-actions.spec.js

```diff
@@ -1,31 +1,24 @@
-(($, _, crmApi, crmStatus, locationService) => {
+(($, _, crmApi, crmConfirm, crmRefreshParent, ts) => {
   describe('Row Actions', () => {
-    let fixture, letterheadId, originalWindow;
+    let fixture, letterheadId;
 
     beforeEach(() => {
-      letterheadId = parseInt(_.uniqueId(), 10);
+      letterheadId = _.uniqueId();
       fixture = `
         <table>
-          <tr data-letterhead-id="${letterheadId}">
+          <tr id="Letterhead-${letterheadId}" class="crm-entity">
```

Comment:
Cant we use the existing `data-letterhead-id` here and in the original place?

Reply:
No, CiviCRM uses this format for the `enable-disable` functionality. This was changed for those row actions in #7 but didn't update this test because we didn't continue working on the delete action and left it for this PR.

I don't like this approach, but they use it internally to know the entity name + the ID of the record 🤷‍♂️